src/ts/features/block-quotes.ts

```diff
@@ -0,0 +1,51 @@
+import {Feature, Settings} from '../utils/settings'
+
+export const config: Feature = {
+    id: 'blockquotes',
+    name: 'Blockquote Support',
+    settings: [{type: 'string', id: 'blockquote_prefix', initValue: '> ', label: 'Blockquote Prefix'}],
+    description:
+        'If you start a block with these characters, custom styles will be added to show the block as a blockquote.',
+}
+
+async function blockquotes() {
+    const blocks = document.querySelectorAll('div.roam-block')
+    for (const block of blocks as NodeListOf<Element>) {
+        const span = block.querySelector('span')
+        const blockquote_prefix = await Settings.get(config.id, 'blockquote_prefix')
+        if (span !== null && span.innerText.trim().startsWith(blockquote_prefix)) {
+            block.classList.add('custom-blockquote')
+        } else {
+            block.classList.remove('custom-blockquote')
+        }
+    }
+    return true
+}
+
+const applyCSS = function () {
```

Comment:
reuse function from `custom-css` feature (probably adding optional id parameter). probably good idea to extract it into a separate utils file too